LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2018 Benoit Lemaire
+Copyright 2020 WalmartLabs
```

Comment:
Can we use `Walmart Labs` here? I know we still have tons of legacy code with the "twitter-style" spelling, which we should also migrate at some point..

Reply:
Yes I'll update it, just copy/pasted from electrode-native one ;)